RxWebKit/Sources/Rx+WebKit.swift

```diff
@@ -60,4 +60,24 @@ extension Reactive where Base: WKWebView {
         return self.observeWeakly(Bool.self, "canGoForward")
             .map { $0 ?? false }
     }
+    
+    
+    /// Reactive wrapper for `evaluateJavaScript(_:completionHandler:)` method.
+    ///
+    /// - Parameter javaScriptString: The JavaScript string to evaluate.
+    /// - Returns: Observable sequence of result of the script evaluation.
+    public func evaluateJavaScript(_ javaScriptString:String) -> Observable<Any> {
+        return Observable.create { [weak base] observer in
+            base?.evaluateJavaScript(javaScriptString) { value, error in
+                if let error = error {
+                    observer.onError(error)
+                } else if let value = value {
+                    observer.onNext(value)
+                } else {
+                    observer.onNext(())
```

Comment:
Let's not do this - I prefer going with `Any?` like the original method and let the user handle it.

Reply:
@bobgodwinx what do u think?

Reply:
change to `Any?` and forgot to add `onCompleted` 😂